src/main/java/com/microsoft/sqlserver/jdbc/SQLServerPreparedStatement.java

```diff
@@ -63,6 +63,9 @@
     /** SQL statement with expanded parameter tokens */
     private String preparedSQL;
 
+    /** True if this execute has been called for this statement at least once */
```

Comment:
Nit: You can use `//` for single line comment instead of using a block comment construct `/* */`